src/api/index.spec.tsx

```diff
@@ -359,4 +349,80 @@ describe(__filename, () => {
       expect(getApiHost({ apiHost, useInsecureProxy: false })).toEqual(apiHost);
     });
   });
+
+  describe('makeApiURL', () => {
+    it('constructs an API URL for a given path', () => {
+      const apiHost = 'https://example.org';
+      const path = '/foo/';
+      const _getApiHost = jest.fn().mockReturnValue(apiHost);
+
+      expect(makeApiURL({ _getApiHost, path })).toEqual(
+        `${apiHost}/api/${defaultVersion}${path}`,
+      );
+    });
+
+    it('can omit the version', () => {
+      const path = '/foo/';
+
+      expect(makeApiURL({ path, version: null })).toEqual(
+        expect.stringMatching(`/api${path}`),
+      );
+    });
+
+    it('can omit the prefix', () => {
+      const path = '/foo/';
+      // This is needed because `stringMatching` won't throw if `prefix` is there.
```

Comment:
I guess you are trying to prevent someone from using `stringMatching`? I don't really see how this comment is helpful.